action.yml

```diff
@@ -76,6 +77,13 @@ runs:
         token: ${{ fromJSON(github.event.inputs.client_payload).payload.github_token }}
         path: "code/"
 
+    - name: Exit if checkout failure is fatal
+      shell: bash
+      if: steps.checkout-original-repo.outcome == 'failure' && fromJSON(github.event.inputs.client_payload).context.job.job_name != 'enrich'
+      run: |
+        echo "Original repo checkout failed, can't continue job execution"
+        exit 1
```

Comment:
maybe worth think of a way to complete the execution on such case?
maybe call complete (and i think we would also need to do some treatment so it won't wait for findings to be uploaded)

Reply:
I don't want to tackle this behavior without proper design and thinking about this, I think for now this is a good solution and we can discuss other controls behavior.
The main usecase for this is enrichment on empty repos, the rest should be very rare and we didn't even see happening.